### ISSUE TITLE
성능 출력 스크립트 작성

### ISSUE THREAD

**Issue by happyhj** [bosspiglet/tensorflow#3]

**Comment by happyhj** [comment]
아래와 같이 사용하면된다
```
python test.py --start_checkpoint=/tmp/speech_commands_train/005/conv.ckpt-18000
```
화면에 validation set  과  test  set  에 대한 정확도와 confusion matrix가 출력된다.
